### PR TITLE
Adding minishift-unstable for minishift 1.0.0-beta.1

### PR DESCRIPTION
**Pull request by tenfourty**
A homebrew cask already exists for `minishift` however we also needed a version for the latest unstable version, hence this pull request to add `minishift-unstable` to homebrew cask versions.

After making all changes to the cask:

- [x] `brew cask audit --download minishift-unstable` is error-free.
- [x] `brew cask style --fix minishift-unstable` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-versions/pulls
[closed issues]: https://github.com/caskroom/homebrew-versions/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/minishift-unstable.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+cask 'minishift-unstable' do
+  version '1.0.0-beta.1'
+  sha256 '2d4020615bd86b814c678fedbf6072488101c26298a90fb081904c6913d88497'
+
+  url "https://github.com/minishift/minishift/releases/download/v#{version}/minishift-#{version}-darwin-amd64.tgz"
+  appcast 'https://github.com/minishift/minishift/releases.atom', checkpoint: 'f78e6fe63a0e0f8f3cae4f97daebe8528e8a38a7d023c6630598b128ee6c8fed'
+  name 'Minishift'
+  homepage 'https://github.com/minishift/minishift'
+
+  depends_on arch: :x86_64
```
Comment (vitorgalvao) [inline Casks/minishift-unstable.rb]: Remove this. It’s useless.

File: Casks/minishift-unstable.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  depends_on formula: 'unar'
```
Comment (vitorgalvao) [inline Casks/minishift-unstable.rb]: It does? Why?

File: Casks/minishift-unstable.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  container type: :tar
```
Comment (vitorgalvao) [inline Casks/minishift-unstable.rb]: Unlikely this is necessary.

File: Casks/minishift-unstable.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
+cask 'minishift-unstable' do
+  version '1.0.0-beta.1'
+  sha256 '2d4020615bd86b814c678fedbf6072488101c26298a90fb081904c6913d88497'
+
+  url "https://github.com/minishift/minishift/releases/download/v#{version}/minishift-#{version}-darwin-amd64.tgz"
+  appcast 'https://github.com/minishift/minishift/releases.atom', checkpoint: 'f78e6fe63a0e0f8f3cae4f97daebe8528e8a38a7d023c6630598b128ee6c8fed'
```
Comment (vitorgalvao) [inline Casks/minishift-unstable.rb]: `checkpoint` needs to be on another line.

File: Casks/minishift-unstable.rb (hunk 3 above)
Comment (tenfourty) [inline Casks/minishift-unstable.rb]: just tested it, as you pointed out this isn't needed so will remove it.

File: Casks/minishift-unstable.rb (hunk 1 above)
Comment (tenfourty) [inline Casks/minishift-unstable.rb]: hi @vitorgalvao do you mean the `name` and `homepage` stuff is useless?

File: Casks/minishift-unstable.rb (hunk 2 above)
Comment (tenfourty) [inline Casks/minishift-unstable.rb]: I'll remove the `arch` and `depends_on` stuff, agreed it's not needed.

File: Casks/minishift-unstable.rb (hunk 4 above)
Comment (tenfourty) [inline Casks/minishift-unstable.rb]: great, will fix this.

**Comment by tenfourty** [comment]
Hi @vitorgalvao thanks for the code review - I've made the changes you requested. I've left in the `name` and `homepage` labels for now.

**Comment by vitorgalvao** [comment]
> I've left in the `name` and `homepage` labels for now.

What do you mean with “for now”? Those are mandatory.

**Comment by tenfourty** [comment]
Awesome, thanks for the help!

**Pull request merged**